src/Lecture3.hs

```diff
@@ -52,15 +52,16 @@ data Weekday
     | Friday
     | Saturday
     | Sunday
-    deriving (Show, Eq)
+    deriving (Show, Eq, Enum, Bounded)
 
 {- | Write a function that will display only the first three letters
 of a weekday.
 
 >>> toShortString Monday
 "Mon"
 -}
-toShortString = error "TODO"
+toShortString :: Weekday -> [Char]
+toShortString day = take 3 (show day)
```

Comment:
Absolutely correct implementation 👍🏻 
You can use function composition and eta-reduce it:
```suggestion
toShortString = take 3 . show
```